Game.cpp

```diff
@@ -0,0 +1,95 @@
+// C++ program to implement tic tac toe game 
+#include <iostream> 
+using namespace std; 
+  
+// Function to draw the Tic-Tac-Toe board 
+void drawBoard(char board[3][3]) 
+{ 
+    cout << "-------------\n"; 
```

Comment:
Avoid using `using namespace std;` as it can lead to namespace pollution. It is better to use `std::` prefix before standard library objects.